mod-circulation/src/main/resources/vega/mod-circulation/eureka-features/root.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Root feature that runs all other mod-circulation features
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+    * def materialTypeId = call uuid1
+    * def materialTypeName = 'e-book'
+    * def requestPolicyIdForGroup = call uuid1
+    * def requestPolicyIdForGroup2 = call uuid1
+    * def requestPolicyIdForGroup3 = call uuid1
+    * def requestPolicyIdForGroup4 = call uuid1
+    * def extRequestTypesForFirstUserGroupRequestPolicy = ["Hold", "Recall"]
+    * def extRequestTypesForSecondUserGroupRequestPolicy = ["Page", "Recall"]
+    * def extRequestTypesForThirdUserGroupRequestPolicy = ["Page", "Hold"]
+    * callonce read('classpath:vega/mod-circulation/eureka-features/util/initData.feature@PostMaterialType') { extMaterialTypeId: #(materialTypeId) }
+#
+#    # groups
+    * def firstUserGroupId = '188f025c-6e52-11ec-90d6-0242ac120003'
+    * def secondUserGroupId = 'f1a28f58-702d-48fe-b95d-daf7fd55dc27'
+    * def thirdUserGroupId = '0dfcce3e-6fb3-11ec-90d6-0242ac120003'
+    * def fourthUserGroupId = 'a58053e4-6fbc-11ec-90d6-0242ac120001'
```

Comment:
I see that we are satisfied with the first three lines without changes, but we changed the fourth. Why?